src/domains/ecommerce-app/api/ecommerce-checkout/v2/_openapi.json.tpl

```diff
@@ -140,7 +140,91 @@
           }
         }
       }
-    }
+    },
+    "/payment-methods/{id}/fees": {
```

Comment:
add bearer authorization only for this api since it's protected by jwt token

Reply:
Solved by https://github.com/pagopa/pagopa-infra/pull/1973/commits/d73aa00abb1b4d4abf5422973ab2945f54bca71a